docs/src/layouts/AppHeader.tsx

```diff
@@ -36,11 +37,24 @@ const Header = styled('header')(({ theme }) => [
 
 const HEIGHT = 56;
 
+const getNameByRoute = (currentRoute: String) => {
+  switch (true) {
+    case currentRoute === 'core':
+      return 'material-ui';
+    case currentRoute === 'x':
+    case currentRoute === 'design-kits':
+    case currentRoute === 'tool-pad':
+      return `mui-${currentRoute}`;
+    default:
+      return '';
+  }
+};
+
 export default function AppHeader() {
   const changeTheme = useChangeTheme();
   const [mode, setMode] = React.useState<string | null>(null);
   const prefersDarkMode = useMediaQuery('(prefers-color-scheme: dark)');
-
+  const repositoryLink = getNameByRoute(useRouter().pathname.substring(1));
```

Comment:
`import { pathnameToLanguage } from 'docs/src/modules/utils/helpers';` serves this purpose

Reply:
Alright, but that seems to just return the current path. I still need to pass that to getNameByRoute to get the proper repository name right? Or am I missing something here.